client/src/modules/Schedule/components/FilteredTags/FilteredTags.tsx

```diff
@@ -0,0 +1,21 @@
+import { Tag } from 'antd';
+import { FilterFilled } from '@ant-design/icons';
+import { TAG_NAME_MAP } from 'modules/Schedule/constants';
+import { CourseScheduleItemDto } from 'api';
+
+type FilteredTagsProps = {
+  tagFilter: string[];
+  onTagClose: (tag: string) => void;
+};
+
+export const FilteredTags = ({ tagFilter, onTagClose }: FilteredTagsProps) =>
+  tagFilter?.length > 0 ? (
+    <div style={{ marginBottom: 16 }}>
+      <FilterFilled style={{ color: 'rgba(0, 0, 0, 0.25)', marginRight: 8 }} />
+      {tagFilter.map(tag => (
+        <Tag key={tag} closable onClose={() => onTagClose(tag)}>{`Tag: ${
+          TAG_NAME_MAP[tag as CourseScheduleItemDto['tag']] || tag
```

Comment:
It would be nice to avoid typecast here, if it's not too complicated, of course :)